examples/jupyter/README.md

```diff
@@ -0,0 +1,17 @@
+# Jupyter 
+It is rather simple to deploy Jupyter on MANTL. There are some official Docker images that you can use for this purpose (https://github.com/jupyter/docker-stacks). For this example we use the jupyter/minimal-notebook.
+
+
+We wrapped the Marathon submit REST call in a small script: `deploy.sh`. You can use it to deploy Jupyter on your MANTL cluster.
+
+```bash
+./deploy.sh
+```
+
+If everything went fine, you should be able to figure out the front end URL of your Jupyter deployment from the Traefik UI.
```

Comment:
It might be worth mentioning that this information is also available in the Marathon UI